htdocs/accountancy/admin/account.php

```diff
@@ -411,42 +431,48 @@
 		print '<td class="liste_titre">&nbsp;</td>';
 	}
 	print '<td class="liste_titre maxwidthsearch">';
-	$searchpicto = $form->showFilterAndCheckAddButtons($massactionbutton ? 1 : 0, 'checkforselect', 1);
+	$searchpicto = $form->showFilterAndCheckAddButtons(0, 'checkforselect', 1);
```

Comment:
Are you sure it is not 1 instead of 0 into first param of showFilterAndCheckAddButtons() ?

Reply:
it's 0 because this is produce if it's 1
![image](https://user-images.githubusercontent.com/45882981/113675942-2e50d700-96bc-11eb-911f-7c54295c8349.png)

Reply:
0 produce 
![image](https://user-images.githubusercontent.com/45882981/113676044-47598800-96bc-11eb-84ce-6071506795e3.png)

Reply:
According to template into modulebuilder/template/myobject_list.php, you must use here
$searchpicto = $form->showFilterButtons()
showFilterAndCheckAddButtons is no more used now.

Reply:
For en enhancement, use CLOSE #... or NEW #... instead of FIX #...